app/src/main/java/com/example/wrk/ScratchCreateAdapter.java

```diff
@@ -64,7 +71,7 @@ public ViewHolder(@NonNull View itemView) {
         public void bind(WorkoutComponent component) {
             Context tableContext = tlComponents.getContext();
             tvExerciseName.setText(component.getExercise().getName());
-            for (int i = 0; i < component.getSets(); i++) {
+            for (int i = 1; i <= component.getSets(); i++) {
```

Comment:
sorry could you explain what changed with bumping i by 1? I don't see it used anywhere

Reply:
It still iterates the same number of times. The reason I have it start at 1 is so when I declare the set number in the textview, it doesn't start with set 0 but instead set 1. I will paste the line that uses it.

tvSetNum.setText(String.valueOf(i));